### ISSUE TITLE
Add more realistic collision models to Prius.sdf / urdf

### ISSUE THREAD

**Issue by liangfok** [RobotLocomotion/drake#2450]
Currently, the collision models are only on the wheels. Ideally, the body of the vehicle so it won't be possible to drive through a concrete column as shown below:

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/w9pJrrAPCNc/0.jpg)](https://www.youtube.com/watch?v=w9pJrrAPCNc)

**Comment by jwnimmer-tri** [comment]
@rpoyner-tri I think perhaps a collision-box model here could be helpful?  I don't think the full body mesh would be. Any advice based on your other work?

**Comment by SeanCurtis-TRI** [comment]
Would we want something in the middle?  Some convex shape that is a tighter fit than a box, but still an order of magnitude simpler than the visualization mesh?

**Comment by liangfok** [comment]
That would be ideal. What app would you recommend using to create such a convex shape? Blender?

**Comment by SeanCurtis-TRI** [comment]
Blender is one option; it's what I did for the iiwa.   I'm more proficient in Maya (attribute that to night on to twenty years of experience) and Thor has a license that may be briefly moochable.  Either way, it's maybe a 30 minute task.

**Comment by jwnimmer-tri** [comment]
I'm not entirely sure what purpose that would serve.  We don't need to simulate body collisions, just acknowledge when first contact occurs.  A conservative box is likely fine for that.

**Comment by liangfok** [comment]
Yes, a conservative box will likely be OK for most situations, and could be a first-spiral solution.
